### ISSUE TITLE
Permission Issue

### ISSUE THREAD

**Issue by Chaaang** [andresperezmelo/niimbot_label_printer#2]
Hello! This function doesn't prompt anything. @andresperezmelo
error " Permission bluetooth granted is false, check in settings that the permission of nearby devices is activated"

![image](https://github.com/user-attachments/assets/ee2cb884-df82-4d28-88a9-c0a3730d28bd)

**Comment by Chaaang** [comment]
I saw this in the library.
![image](https://github.com/user-attachments/assets/d3e98098-b1a5-4546-aa98-3c719e465e27)
if sdkversion is >= 31 permission is granted. But my sdkversion is 34 and when its not granting permission, I manually allow the application permission on my device. Is there other way to grant permission rather than going to my device setting manually?

**Comment by andresperezmelo** [comment]
If there's another way to grant permission, it's through execution. When validating the permission, if it's false, it can be requested using the package [permission_handler](https://pub.dev/packages/permission_handler). You must add the permission to the Android manifest.

**Comment by Chaaang** [comment]
Yes, when I check for bluetooth permission using permission handler, it returns returns true but I still cant use the package the error message still occur.

**Comment by Chaaang** [comment]
Is there other permission I need to request?
